nx/source/services/hid.c

```diff
@@ -569,8 +570,7 @@ Result hidMergeSingleJoyAsDualJoy(HidControllerID id0, HidControllerID id1) {
 
     rc = appletGetAppletResourceUserId(&AppletResourceUserId);
     if (R_FAILED(rc))
-        return rc;
-
+        AppletResourceUserId = 0;
```

Comment:
The newline was mistakenly removed here.

Reply:
you're right, I'll fix that.